RelaxOn/Views/Home/Music/MusicViewModel.swift

```diff
@@ -184,113 +229,87 @@ final class MusicViewModel: NSObject, ObservableObject {
+                        if isPresented {
+                            self?.playPause()
+                        } else {
+                            self?.initiatedByWatch = true
+                            if (UIApplication.shared.applicationState == .background) {
+                                self?.playPause()
+                            }
+                        }
+                    } else {
+                        self?.playPause()
+                    }
+                    case "prev":
+                        guard let mixedSound = self?.mixedSound else { return }
+                        self?.setupPreviousTrack(mixedSound: mixedSound)
+                    case "next":
+                        guard let mixedSound = self?.mixedSound else { return }
+                        self?.setupNextTrack(mixedSound: mixedSound)
+                    default:
+                        print("해당 안됨")
+                }
             }
-            self.setupNextTrack(mixedSound: mixedSound)
-        default:
-            print("unknown watchinfo")
         }
-    }
-    
-    func session(_ session: WCSession, didReceiveUserInfo userInfo: [String : Any] = [:]) {
-        print("received from watch")
         
-        DispatchQueue.main.async {
-            let key = "watchInfo"
-            guard let WatchInfo = userInfo[key] as? [String] else {
-                return
-            }
-            
-            let index = userRepositories.firstIndex { element in
-                element.name == WatchInfo[0]
+        if let request = message["list"] as? String {
+            DispatchQueue.main.async { [weak self] in
+                self?.sendMessage(key: "list", userRepositories.map{mixedSound in mixedSound.name})
             }
-            
-            self.mixedSound = userRepositories[index ?? 0]
-            
-            switch WatchInfo[1] {
-            case "playing", "paused":
-                print("playPause()")
-                self.playPause()
-            case "prev":
-                print("prev()")
-                guard let mixedSound = self.mixedSound else {
-                    return
-                }
-                self.setupPreviousTrack(mixedSound: mixedSound)
-            case "next":
-                print("next()")
-                guard let mixedSound = self.mixedSound else {
-                    return
+        }
+
+        if let title = message[titleMessageKey] as? String {
+            DispatchQueue.main.async { [weak self] in
+                self?.currentTitle = title
+                if let isPresented = self?.isMusicViewPresented {
+                    if isPresented {
+                        self?.startPlayerFromWatch()
+                    } else {
+                        self?.initiatedByWatch = true
+                        self?.startPlayerFromWatch()
+                    }
+                } else {
+                    self?.startPlayerFromWatch()
```

Comment:
if else 구문이 많아서 가독성이 떨어지는 것 같습니다. 어떤 경우에도 self.?startPlayerFromWatch()가 실행되는 것 같아서 다음과 같이 코드를 짤 수 있을 것 같습니다. 
```suggestion
     guard let title = message[titleMessageKey] as? String else { return }

DispatchQueue.main.async { [weak self] in
    self?.currentTitle = title
    if let isPresented = self?.isMusicViewPresented {
        if !isPresented {
            self?.initiatedByWatch = true
        }
    }
    self?.startPlayerFromWatch()
}
```